setup.cfg

```diff
@@ -0,0 +1,18 @@
+[metadata]
+name = trove-classifiers
+url = https://github.com/pypa/trove-classifiers
+author = The PyPI Admins
+author_email = admin@pypi.org
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Developers
+    License :: OSI Approved :: Apache Software License
+    Programming Language :: Python :: 3
+license_files = LICENSE
+description = Canonical source for classifiers on PyPI (pypi.org).
+long_description = file: README.md
+long_description_content_type = text/markdown
+keywords = classifiers
+
+[options]
+packages = trove_classifiers
```

Comment:
Please change this to use the equivalent syntax for `find_packages()`.

Reply:
I wonder if it's better in this case to be explicit, since there are many folders potentially be misrecognized as packages and it is unlikely this project will have an additional package.  Alternatives include:

* Use `src` package dir
* Explicitly exclude `tests`, `bin`, etc.

Reply:
I'm a huge fan of [using the `src`](https://blog.ionelmc.ro/2014/05/25/python-packaging/#the-structure).

Reply:
e10a6715dfef4238ebc880ec18865ba0b30be492 changes the layout to `src`.  I'll squash the commits together if this is preferred over explicit specification of package.